src/Propel/Generator/Reverse/PgsqlSchemaParser.php

```diff
@@ -347,7 +350,7 @@ protected function isColumnDefaultExpression($default)
             return true;
         }
 
-        false;
+        return false;
```

Comment:
Maybe it should return null and get a bug ticket.

I'd rather not change the previously questionable behaviour of this code.

Reply:
This seems to be a bug. Tests pass so there is some evidence this can be cleaned up now for next alpha.
Bear in mind: we can and should break those bc topics if it fixes things until we release beta and higher.

Reply:
Okay looks fine to me after I looked deeper into the usages